sale_order_product_assortment/tests/test_sale_order_product_assortment.py

```diff
@@ -1,17 +1,19 @@
 # Copyright 2020 Tecnativa - Carlos Roca
+# Copyright 2024 Tecnativa - Carolina Fernandez
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
 from odoo.tests.common import TransactionCase
 
 
 class TestProductAssortment(TransactionCase):
```

Comment:
Speed up a bit the execution of the test with:

```
from odoo.odoo.addons.base.tests.common import BaseCommon
```

and inheriting the class from that one.